NEMO/admin.py

```diff
@@ -111,6 +111,10 @@ class Media:
 		widget=FilteredSelectMultiple(verbose_name="Nonrequired resources", is_stacked=False),
 	)
 
+	_tool_calendar_color = forms.CharField(
+		required=False, max_length=9,
```

Comment:
you need to set the initial value otherwise when you create a new tool it defaults to black and is saved as black in the db
```suggestion
		required=False, max_length=9, initial='#33ad33',
```